tower-test/Cargo.toml

```diff
@@ -27,4 +27,4 @@ futures-executor-preview = "0.3.0-alpha.18"
 tokio-test = "0.2.0-alpha.2"
 tokio-sync = "0.2.0-alpha.2"
 tower-service = "0.3.0-alpha.1"
-pin-utils = "0.1.0-alpha.4"
\ No newline at end of file
+pin-utils = "0.1.0-alpha.4"
```

Comment:
btw pin_mut is avail in futures-util

Reply:
This was there from before, but I've replaced it with `pin-project` now.